apcd-cms/src/apps/extension/templates/extension_submission_form/extension_submission_form.html

```diff
@@ -33,20 +33,18 @@ <h1>Request an Extension</h1>
           <form action="" method="POST">
             {% csrf_token %}
             <h4>Extension Information</h4>
-            
               <div class="field-wrapper textinput required">
                 <div class="field-errors" style="display: none"></div>
-      
                 <label for="extension-type">
                   Extension Type<span class="asterisk">*</span>
                 </label>
-      
-                <input type="text" name="extension-type" class="textinput" id="extension-type" minlength="1" maxlength="50"
-                  required />
-                  <div id="help-text-extension-type" class="help-text">
-                    50 character limit
-                  </div>
-              </div>
+                <div class="field-errors" style="display: none"></div>
+                  <select name='extension-type' required='required' class="choicefield" id='extension-type'>
+                    <option class="dropdown-text" value="regular">Regularly Scheduled Extension</option>
+                    <option class="dropdown-text" value="resubmission">Corrected Resubmission</option>
+                    <option class="dropdown-text" value="small_carrier">Small Carrier (Fewer Than 10,000 Lives Covered)</option>
+                </select>
+                </div>
```

Comment:
These are all awesome! If you have some spare time to bring all the <div> indents in line with their </div>s that would be great, but otherwise this code is super easy to read and test!

Reply:
Good call. Done and pushed up.